samples/README.md

```diff
@@ -0,0 +1,24 @@
+# samples
```

Comment:
notes about the examples:
1. we can call jaxrs/springmvc server implementation with RPC style client code, this is not our intention, but this seams a quite great feature. We can discuss if include this feature in the example. Before that, we must test any other HTTP methods like GET/POST
2. should add client codes use RestTemplate to call jarxs/springmvc service, that's what we have given in other demos.